### ISSUE TITLE
Use dictionary object for airport/region/location lookup

### ISSUE THREAD

**Issue by longhorn09** [longhorn09/aws_prices#2]
Convert `if / elif` code for regional lookup to a streamlined dictionary

```
region_map={
    "CMH": ("us-west-2", "US East (Ohio)"),
    "DUB": ("eu-west-1", "EU (Ireland)"),
    ...
}

def getAWSRegionFromCode(region_code):
    return region_map.get(region_code, (None, None))[0]

def getAWSLocationFromCode(region_code):
    return region_map.get(region_code, (None, None))[1]
```

**Comment by longhorn09** [comment]
```
    region_map={
        "CMH": ("us-east-2", "US East (Ohio)"),
        "DUB": ("eu-west-1", "EU (Ireland)"),
        "FRA": ("eu-central-1","EU (Frankfurt)"),
        "GRU": ("sa-east-1","South America (Sao Paulo)"),
        "IAD": ("us-east-1","US East (N. Virginia)"),
        "LHR": ("eu-west-2","EU (London)"),
        "NRT": ("ap-northeast-1","Asia Pacific (Tokyo)"),
        "PDX": ("us-west-2","US West (Oregon)"),
        "SIN": ("ap-southeast-1","Asia Pacific (Singapore)"),
        "SYD": ("ap-southeast-2","Asia Pacific (Sydney)"),
    }
```

**Comment by longhorn09** [comment]
resolved with commit f83a614bbd82081999bbc99c6602d1eb01b7ada7
